src/plopp/backends/matplotlib/canvas.py

```diff
@@ -112,11 +143,7 @@ def __init__(
         self.ax.set_title(title)
         self.ax.grid(grid)
         self._coord_formatters = []
-
-        self._xmin = np.inf
-        self._xmax = np.NINF
-        self._ymin = np.inf
-        self._ymax = np.NINF
+        self._bbox = BoundingBox()
```

Comment:
Would it be possible to have this only be a local variable in the `autoscale()` method? I saw that it is used in `logy`  but I didn't really understand why.

Reply:
It's also used in `autoscale` itself. It's used as a global memory of the range. It is needed when you are in a mode where you allow the plot limits to grow but not shrink.
The default is that the range follow the data, but you can also set a mode where limits grow, maybe giving a better feel for the relative differences in the data.

See the `autoscale` parameter in https://scipp.github.io/plopp/reference/generated/plopp.plot.html#plopp.plot